src/RouteDescriber/RouteArgumentDescriber/SymfonyMapQueryParameterDescriber.php

```diff
@@ -141,4 +141,35 @@ private function describeValidateFilter(?int $filter, int $flags, array $options
 
         return [];
     }
+
+    private function getEcmaRegexpFromPCRE(string $pcreRegex): string
+    {
+        // Check if PCRE regex has delimiters
+        if (!preg_match('/^(.)(.*)\1([a-zA-Z]*)$/s', $pcreRegex, $matches)) {
+            throw new \InvalidArgumentException('Invalid PCRE regex format. Missing delimiters.');
```

Comment:
Maybe we shouldn't throw here to preserve original behaviour? That way it 'tries its best' to generate documentation.

```suggestion
            return $pcreRegex;
```